### DISCUSSION TITLE
Best practices for updating records?

### DISCUSSION THREAD

**Discussion by heymartinadams** [prisma/prisma#3138] [Q&A] [answered]
What are best practices for running a `prisma.update` command where I just want to update a record without querying any data? My understanding [from the docs](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/crud#update) is that any `update` will query the entire model, unless limited with the `select` command.

It’s probably irrelevant from a performance standpoint, but I still want to not use up any unnecessary resources.

Is this the most effective statement if all I want to do is update a `mobileVerified` field?

```js
await prisma.user.update({
	where: { mobile: '+12345678900 },
	data: {
		mobileVerified: true
	},
	select: {
		id: true
	}
})
```

**Comment by ryands17** [comment, marked as answer]
Hey @heymartinadams 👋 
Yes currently you would need to use select just to query the `id` as a workaround.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
